routes/web.php

```diff
@@ -18,7 +18,13 @@
 
 Route::view('/about', 'about')->name('about');
 
-Route::view('/', 'home/feed')->name('home.feed');
+Route::get('/', function () {
+    return auth()->check()
+        ? redirect(route('home.'.auth()->user()->default_tab)) // @phpstan-ignore-line
```

Comment:
what error it has?

Reply:
Property 'default_tab' not found in Illuminate\Contracts\Auth\Authenticatable|null